src/components/AppMetadata.svelte

```diff
@@ -0,0 +1,33 @@
+<!-- AppDetails.svelte -->
+<script lang="ts">
+    import { createEventDispatcher } from 'svelte';
+
+    import type { PageData } from '../routes/app/[id]/$types';
```

Comment:
This is an inappropriate coupling. It is more appropriate to have separate `export let` properties.  

Also, how will you deal with selecting the correct localization based on the browser language?  Currently, it is hardcoded to `listings[0]`. We can wait on that for a little longer, but it should be scheduled.